Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Software.Updater/Sharp Update/Classes/Hasher.cs

```diff
@@ -1,65 +1,48 @@
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in all
- * copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- * SOFTWARE.
- *
- */
-#endregion
-
-namespace Krypton.Toolkit.Suite.Extended.Software.Updater.SharpUpdate
+﻿namespace Krypton.Toolkit.Suite.Extended.Software.Updater
 {
-    internal class Hasher
+    /// <summary>
+    /// Class used to generate hash sums of files
+    /// </summary>
+    internal static class Hasher
     {
-        #region Methods
-        internal static string HashFile(string filePath, HashType hashType)
+        /// <summary>
+        /// Generate a hash sum of a file
+        /// </summary>
+        /// <param name="filePath">The file to hash</param>
+        /// <param name="algo">The Type of hash</param>
+        /// <returns>The computed hash</returns>
+        internal static string HashFile(string filePath, HashType algo)
         {
-            switch (hashType)
+            switch (algo)
             {
-                case HashType.Md5:
+                case HashType.MD5:
                     return MakeHashString(MD5.Create().ComputeHash(new FileStream(filePath, FileMode.Open)));
-                case HashType.Sha1:
+                case HashType.SHA1:
                     return MakeHashString(SHA1.Create().ComputeHash(new FileStream(filePath, FileMode.Open)));
-                case HashType.Sha256:
+                case HashType.SHA256:
                     return MakeHashString(SHA256.Create().ComputeHash(new FileStream(filePath, FileMode.Open)));
-                case HashType.Sha384:
+                case HashType.SHA384:
                     return MakeHashString(SHA384.Create().ComputeHash(new FileStream(filePath, FileMode.Open)));
-                case HashType.Sha512:
+                case HashType.SHA512:
                     return MakeHashString(SHA512.Create().ComputeHash(new FileStream(filePath, FileMode.Open)));
                 default:
-                    return string.Empty;
+                    return "";
```

Comment:
Use of `string.empty` means that it is on purpose, and not some string will be filled in later sort of thing

Reply:
Will do on next updates